stock_picking_move_description/models/res_config_settings.py

```diff
@@ -0,0 +1,10 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    group_picking_report = fields.Boolean(implied_group="stock_picking_move_description.group_picking_report")
```

Comment:
Add a string label with main group title, e.g. the same as new group description